### PR TITLE
added responsiveness

### PR DESCRIPTION
**Pull request by preethasri**
added a bottom navbar for mobileapp
added responsiveness

### CODE REVIEW

File: src/Components/BottomNav/BottomNav.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    left:0;
+    width:100%;
+    height:4rem;
+    padding:0 1rem;
+    display:grid;
+    align-content: center;
+    border-radius: 1.25rem 1.25rem 0 0;
+    transition:0.4s;
+    box-shadow: rgba(149, 157, 165, 0.2) 0px 8px 24px;
+    background-color: var(--bg-color);
```
Comment (Sharmaryan) [inline src/Components/BottomNav/BottomNav.css:14]: you can remove px after 0

File: src/features/post/components/CommentModal.js [hunk 2]
```diff
@@ -21,21 +21,22 @@ export const CommentModal = ({
...
   const addCommentHandler = (e) => {
     e.preventDefault();
-
-    if(comment.trim()){
-      if(currentComment._id){
-      dispatch(editComment({token,postId:post._id,commentData:{...currentComment,comment},commentId}))
-      }
-    else{
+    console.log("working")
+
```
Comment (Sharmaryan) [inline src/features/post/components/CommentModal.js:30]: remove console log

**Review by Sharmaryan — APPROVED** [review]

File: src/App.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ function App() {
     
   return(
     <>
-   <div className="text-black">
+   <div>
     <MyRoutes />
     </div>
     </>
```
Comment (JowelTisso) [inline src/App.js:12]: * Fragment is not required here, since you are returning a single div

**Review by JowelTisso — APPROVED** [review]
* Clean code 
* Looks Good

**Review by GeetaPogul — COMMENTED** [review]
looks good and clean

**Pull request merged**